doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -4189,6 +4305,24 @@ MCUboot
 
 The issues in this section are related to :ref:`MCUboot <mcuboot_wrapper>`.
 
+.. rst-class:: v3-1-0
+
+NCSDK-34594: MCUboot might hang and is failing on FPROTEC assertion when using external flash
+  FPROTEC can support locking up to 62 kB of RRAM on nRF54L SoCs.
+  Either the partition declared for MCUboot or its size might be too large for using FPROTECT (depends on MCUboot configuration).
+  Without access to MCUboot, this might be misrecognized by an application.
```

Comment:
Without access to MCUboot Logs, this might be misrecognized as not working/starting application.